src/CoreLocation/CLEnums.cs

```diff
@@ -67,6 +67,10 @@ public enum CLError : long {
 
 		// ios14
 		PromptDeclined = 18,
+
+		// ios16
+		[NoMac, iOS (16,1), MacCatalyst (16,0), Watch (9,1), TV (16,1)]
```

Comment:
Is it MacCatalyst (16,1) instead of (16,0)?

Reply:
```suggestion
		[NoMac, iOS (16,1), MacCatalyst (16,1), Watch (9,1), TV (16,1)]
```